revproxy/utils.py

```diff
@@ -3,31 +3,42 @@
 
 import logging
 
-
+#: List containing string constant that are used to represent headers that can
+#: be ignored in the required_header function
 IGNORE_HEADERS = (
     'HTTP_ACCEPT_ENCODING',  # We want content to be uncompressed so
-                             #   we remove the Accept-Encoding from
-                             #   original request
+                             # we remove the Accept-Encoding from
```

Comment:
Why did you take the indentation out? This makes easier to see that the comment continues and shouldn't break any pep8 rules.